### DISCUSSION TITLE
help - get_page_info

### DISCUSSION THREAD

**Discussion by JillieChang** [kevinzg/facebook-scraper#806] [Q&A] [answered]
Hi, 

Please, could you tell me why when I run the following code
  
    pprint(get_page_info("pages/Giraffas/202339333139400"))

instead of get 

  {'checkins': 223, 'likes': 63, 'people_talking_about_this': 1}

I get this
 
{'likes': 63,
 'name': 'Giraffas - Rio de Janeiro, RJ, Brazil - Restaurant | Facebook\n'
         'requireLazy(["HasteSupportData"],function(m){m.handle({"clpData":{"1814852":{"r":1},"1838142":{"r":1,"s":1},"1744166":{"r":1,"s":1},"1949898":{"r":1},"1848815":{"r":10000,"s":1},"1744178":{"r":1,"s":1}},"gkxData":{"708253":{"result":false,"hash":"AT5n4hBL3YTMnQWt3Ho"},"1167394":{"result":false,"hash":"AT7BpN-tlUPwbIIF7rs"},"5241":{"result":true,"hash":"AT7o1bCQPGpf3ShEGPU"},"1073500":{"result":false,"hash":"AT7aJmfnqWyioxOO3Ek"},"1224637":{"result":false,"hash":"AT7JRluWxuwDm3XzLMM"},"5919":{"result":false,"hash":"AT6PGptIPUYH051BwhI"},"676837":{"result":false,"hash":"AT4N8wBZA8ctCdHwBu8"},"996940":{"result":false,"hash":"AT7opYuEGy3sjG1aGx8"},"1263340":{"result":false,"hash":"AT5bwizWgDaFQu ...

the same happens with any ID. I get disordered information.

Thanks

**Comment by neon-ninja** [comment, marked as answer]
Try update lxml with `pip install -U lxml`
